test/views/HomePageViewSpec.scala

```diff
@@ -245,14 +246,23 @@ class HomePageViewSpec extends TestSupport {
 
     "have a payment history tile" which {
       "has a heading" in new Setup {
-        getElementById("payment-history-tile").map(_.select("h2").text) shouldBe Some(messages("home.paymentHistory.heading"))
+        getElementById("payment-history-tile").map(_.select("h2").text) shouldBe Some(messages("home.paymentHistoryRefund.heading"))
```

Comment:
there should be 2 tests: one for when the feature switch is enabled, and one where it is disabled and we should see the value changing..